src/components/navigation/container/navigation.scss

```diff
@@ -72,7 +73,7 @@
     .link {
         > a {
             display: block;
-            padding: 17px 15px 0 15px;
+            padding: 15px 15px 2px 15px;
```

Comment:
Text looks miss-aligned vertically. Can we update to `padding: 13px 15px 3px 15px`